docs/validations.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+import Textbox from "../src/components/textbox";
+import DateRange from "../src/components/date-range";
+import { RadioButton, RadioButtonGroup } from "../src/components/radio-button";
+
+/**
+ * This file is used primarily as a means to generate the props table.
+ * It contains the tag: ["isHidden"] so that it is not included in the sidebar.
+ */
+
+const meta: Meta = {
+  title: "Documentation/Validations",
+  tags: ["isHidden"],
+};
+
+export default meta;
+type Story = StoryObj;
```

Comment:
**comment** (non-blocking): I'm not sure the saving of 3 characters a few times over is worth renaming this type for - but I don't object if you prefer it this way

Reply:
It's a fair point. It was taken from the CSF documentation on the Storybook site but it is missing the `typeOf meta` part which it should really have I believe. This will need doing across all `stories.tsx` files, unfortunately.